pkg/controller/clusterrelocate/clusterrelocate_controller.go

```diff
@@ -521,14 +522,18 @@ func (r *ReconcileClusterRelocate) copy(cd *hivev1.ClusterDeployment, destClient
 		return errors.Wrap(err, "could not get DNSZone")
 	}
 	if dnsZone != nil {
+		logger = logger.WithField("type", reflect.TypeOf(dnsZone)).WithField("resource", dnsZone.Name)
 		if err := r.copyResource(dnsZone, destClient, false, logger); err != nil {
 			return errors.Wrap(err, "failed to copy dnszone")
 		}
 	}
 
 	// copy clusterdeployment
-	if err := r.copyResource(cd, destClient, true, logger); err != nil {
-		return errors.Wrap(err, "failed to copy clusterdeployment")
+	{
```

Comment:
To keep the `logger` variable isolate to inside the scope, in case there is other code added after the code block later that may use `logger` and not want the type and resource field. But, now I see that I replaced the function-scoped `logger` variable instead of creating a new `logger` variable.